src/common/services/recordToSchemaMapping/recordToSchemaMapping.service.ts

```diff
@@ -246,9 +243,8 @@ export class RecordToSchemaMappingService implements IRecordToSchemaMapping {
   private hasCorrectUuid(entry: SchemaEntry, recordKey: string) {
     const isUIControl = UI_CONTROLS_LIST.includes(entry.type as AdvancedFieldTypeEnum);
     const hasTheRecordUri = entry.uriBFLite === recordKey;
-    const hasTheMappedFieldUri = entry.uriBFLite === recordKey;
```

Comment:
@SKarolFolio These appear to have been different when I looked at earlier revisions, but they're equivalent now. Any need to keep both?